elements/lisk-cryptography/test/encrypt.spec.ts

```diff
@@ -188,48 +186,52 @@ describe('encrypt', () => {
 			});
 
 			it('should encrypt a passphrase', () => {
-				return expect(encryptedPassphrase).to.have.property('cipherText').and.be
-					.hexString;
+				// TODO: Add Hex string assertion check
+				expect(encryptedPassphrase).toMatchObject({
+					cipherText: expect.any(String),
+				});
+				expect(encryptedPassphrase).toHaveProperty('cipherText');
+				expect(regHexadecimal.test(encryptedPassphrase.iv)).toBe(true);
+				expect(encryptedPassphrase.iv).toHaveLength(24);
 			});
 
 			it('should output the IV', () => {
-				return expect(encryptedPassphrase)
-					.to.have.property('iv')
-					.and.be.hexString.and.have.length(24);
+				expect(encryptedPassphrase).toHaveProperty('iv');
+				expect(encryptedPassphrase.iv).toHaveLength(24);
 			});
 
 			it('should output the salt', () => {
-				return expect(encryptedPassphrase)
-					.to.have.property('salt')
-					.and.be.hexString.and.have.length(32);
+				expect(encryptedPassphrase).toHaveProperty('salt');
+				expect(encryptedPassphrase.salt).toHaveLength(32);
 			});
 
 			it('should output the tag', () => {
-				return expect(encryptedPassphrase)
-					.to.have.property('tag')
-					.and.be.hexString.and.have.length(32);
+				expect(encryptedPassphrase).toHaveProperty('tag');
+				expect(encryptedPassphrase.tag).toHaveLength(32);
 			});
 
 			it('should output the current version of Lisk Elements', () => {
-				return expect(encryptedPassphrase)
-					.to.have.property('version')
-					.which.is.equal(ENCRYPTION_VERSION);
+				expect(encryptedPassphrase).toHaveProperty(
+					'version',
+					ENCRYPTION_VERSION,
+				);
 			});
 
 			it('should output the default number of iterations', () => {
-				return expect(encryptedPassphrase)
-					.to.have.property('iterations')
-					.equal(PBKDF2_ITERATIONS);
+				expect(encryptedPassphrase).toHaveProperty(
+					'iterations',
+					PBKDF2_ITERATIONS,
+				);
 			});
 
-			it('should take more than 0.5 seconds @node-only', () => {
+			it('should take more than 0.25 seconds @node-only', () => {
```

Comment:
why is this changing? @valamidev

Reply:
Test are now executing faster, so it finish in a ~450-500ms range. Seems it depend from CPU speed, so I do allow twice faster execution to avoid test fail on high-end CPUs.

Reply:
These 2 tests are removed, due invalid time/performance measurement.